rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_fan_configs_parallel.py

```diff
@@ -0,0 +1,31 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.ruleset_functions.baseline_systems.baseline_system_util import (
+    find_exact_one_terminal_unit,
+)
+
+TERMINAL_FAN_CONFIGURATION = schema_enums["TerminalFanConfigurationOptions"]
+
+
+def are_all_terminal_fans_config_parallel(rmi_b, terminal_unit_id_list):
```

Comment:
The function name and the PR name don't match. But, I believe the function name is correct.

Further, the name of the [RDS ](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_fan_configs_parallel.md#:~:text=Blame-,are_all_terminal_fan_configs_parallel,-Description%3A%20Returns)is ```are_all_terminal_fan_config**s**_parallel``` instead of ```are_all_terminal_fans_config_parallel```. Could you double-check this?